docs/adr/0032-l10n-i18n-improvements.md

```diff
@@ -0,0 +1,138 @@
+
+### Pros of dropping fallback text
+
+- Better maintainability and guarantee of consistency due to not needing duplicate strings (one for fallback text and one in an FTL file), in the codebase
+- It might feel more i18n-friendly to depend on Fluent instead of prioritizing English speaking users by explicitly setting English fallback text
+
+### Cons of dropping fallback text
+
+- String extraction timing considerations; there is currently a delay between when FxA lands PRs and when we can reference the latest compiled FTL files, including `en`
+- `@fluent/react` docs actually [recommend fallback text](https://github.com/projectfluent/fluent.js/wiki/Localized#fallback-string) and state that in the future, it may be used for automatic [extraction of source copy](https://github.com/projectfluent/fluent.js/wiki/Localized#fallback-string)
+- We ran a small test (see technical story doc) that showed at least on rare occasions, Fluent may fail to retrieve strings
+- It would take at least two searches to find where a string should ultimately end up in a component, e.g. finding the corresponding FTL ID then searching for it
+
+### Solutions
+
+One of the primary concerns around using fallback text is maintainability and consistency guarantee. We should be testing:
+
+- All provided FTL IDs exist in the source-of-truth `en` bundle
+- Fallback text matches the message tied to the ID in the `en.ftl` bundle, and validate that variables referenced in the message string are available
+- The message doesn’t contain straight quotes (nice to have)
+
+#### React-based packages
+
+We will create two wrappers, one for `Localized` and one for `l10n.getString()`, to enforce fallback text is always provided, set up a `merge:ftl-test` script for when tests run, and mock the wrappers to test what's outlined above.
+
+#### `fxa-auth-server`
+
+We will create a helper function to send into our templates to more easily grab messages from Fluent bundles for things like alt text that otherwise require additional complexities due to our MJML and `@fluent/dom` set up. We can also automatically apply `data-l10n-args`.
+
+We test our email templates in an enormous, single test file of email strings. We will adjust our email test set up and separate these out per component for better maintainability and easier testing.
+
+## Decision outcome 3: concat `fxa-react` FTL files to other packages, use a global branding file
+
+When we use components from `fxa-react` in another package, we have to add the FTL ID and string into the package using the component rather than in `fxa-react`. This can be easily forgotten or lead to duplication of IDs/messages across our packages.
+
+Instead, we can concat files from `fxa-react` into the existing React bundles. Additionally, to help with consistency and maintainability across the board, we'll create a "global" FTL file with our license and branding to be shared across our packages to be used first to build our bundles.
```

Comment:
Did you mean "Fluent bundles" instead of "React bundles"?

Reply:
I did, thanks! 🙇‍♀️